site/src/docs/components/header/index.mdx

```diff
@@ -63,3 +63,18 @@ Coming soon!
 #### With navigation bar
 
 Coming soon!
+
+#### With skip link
+
+- Using the Header.SkipLink component is optional. Use the skip link component to help keyboard-only users skip to the main content on a page.
+
+<PlaygroundPreview>
+  <Header>
+    <Header.SkipLink skipTo="#content" label="Skip to content"/>
+    <Header.UniversalBar primaryLinkHref="#" primaryLinkText="Helsingin kaupunki">
+      <Header.NavigationLink href="#">Link</Header.NavigationLink>
+      <Header.NavigationLink href="#">Link</Header.NavigationLink>
+      <Header.NavigationLink href="#">Link</Header.NavigationLink>
+    </Header.UniversalBar>
```

Comment:
Could you add ActionBar and NavigationMenu here too so there's more "meat" to the example?